2024-12-17 19:06:05,346 - asyncio - DEBUG - Using proactor: IocpProactor
2024-12-17 19:06:05,349 - root - INFO - Searching for GoPro cameras...
2024-12-17 19:06:05,487 - root - INFO - Discovered GoPro: C3461326143885._gopro-web._tcp.local. at 172.28.185.51
2024-12-17 19:06:05,498 - root - INFO - Discovered GoPro: C3461350386740._gopro-web._tcp.local. at 172.27.140.51
2024-12-17 19:06:10,356 - root - INFO - Found the following GoPro devices:
2024-12-17 19:06:10,356 - root - INFO - Name: C3461326143885._gopro-web._tcp.local., IP: 172.28.185.51
2024-12-17 19:06:10,359 - root - INFO - Name: C3461350386740._gopro-web._tcp.local., IP: 172.27.140.51
2024-12-17 19:06:10,360 - root - INFO - Cache updated successfully with newly discovered devices.
2024-12-17 19:06:10,361 - root - INFO - Resetting and enabling USB control on all cameras...
2024-12-17 19:06:10,362 - root - INFO - Resetting USB control on camera 172.28.185.51.
2024-12-17 19:06:10,364 - root - INFO - Resetting USB control on camera 172.27.140.51.
2024-12-17 19:06:10,374 - urllib3.connectionpool - DEBUG - Starting new HTTP connection (1): 172.27.140.51:8080
2024-12-17 19:06:10,377 - urllib3.connectionpool - DEBUG - Starting new HTTP connection (1): 172.28.185.51:8080
2024-12-17 19:06:10,435 - urllib3.connectionpool - DEBUG - http://172.27.140.51:8080 "GET /gopro/camera/control/wired_usb?p=0 HTTP/11" 200 3
2024-12-17 19:06:10,436 - root - INFO - USB control disabled on camera 172.27.140.51.
2024-12-17 19:06:10,466 - urllib3.connectionpool - DEBUG - http://172.28.185.51:8080 "GET /gopro/camera/control/wired_usb?p=0 HTTP/11" 200 3
2024-12-17 19:06:10,467 - root - INFO - USB control disabled on camera 172.28.185.51.
2024-12-17 19:06:12,439 - urllib3.connectionpool - DEBUG - Starting new HTTP connection (1): 172.27.140.51:8080
2024-12-17 19:06:12,471 - urllib3.connectionpool - DEBUG - Starting new HTTP connection (1): 172.28.185.51:8080
2024-12-17 19:06:12,525 - urllib3.connectionpool - DEBUG - http://172.27.140.51:8080 "GET /gopro/camera/control/wired_usb?p=1 HTTP/11" 200 3
2024-12-17 19:06:12,525 - root - INFO - USB control enabled on camera 172.27.140.51.
2024-12-17 19:06:12,540 - urllib3.connectionpool - DEBUG - http://172.28.185.51:8080 "GET /gopro/camera/control/wired_usb?p=1 HTTP/11" 200 3
2024-12-17 19:06:12,541 - root - INFO - USB control enabled on camera 172.28.185.51.
2024-12-17 19:06:12,541 - root - INFO - Synchronizing time on all cameras...
2024-12-17 19:06:12,542 - root - ERROR - Failed to sync time: cannot import name 'sync_time' from 'date_time_sync' (c:\!AndriiShramko\inventions\SDKGOPROShramko\GoProControl\api\v2\v39\date_time_sync.py)
2024-12-17 19:06:12,542 - root - ERROR - An error occurred: cannot import name 'sync_time' from 'date_time_sync' (c:\!AndriiShramko\inventions\SDKGOPROShramko\GoProControl\api\v2\v39\date_time_sync.py)
